plugins/com.google.cloud.tools.eclipse.appengine.newproject/src/com/google/cloud/tools/eclipse/appengine/newproject/CreateAppEngineStandardWtpProject.java

```diff
@@ -76,6 +77,10 @@ IFile getMostImportant() {
     }
     this.config = config;
     this.uiInfoAdapter = uiInfoAdapter;
+
+    // Workaround deadlock bug described in Eclipse bug (https://bugs.eclipse.org/511793)
+    // by ensuring activation of the org.eclipse.jst.j2ee bundle
+    IJ2EEFacetConstants.DYNAMIC_WEB.toLowerCase();
```

Comment:
So this can happen outside our tests, right? I wonder if this bundle is loaded (activated) all the time whenever a user uses our plugin. If not, will it be better to a more fundamental bundle such as `.appengine.facets`?